sympy/calculus/tests/test_util.py

```diff
@@ -492,6 +494,177 @@ def test_AccumBounds_pow():
         Pow(AccumBounds(-oo, oo), AccumBounds(0, 1))
 
 
+def test_AccumBounds_as_exponent():
+    B = AccumBounds
+    # base = -1
+    assert (-1)**B(-2, -1) == B(-1, 1)  # even and odd
```

Comment:
`(-1)**x` is not real for `x` in the interval `[-2. -1]` (or any other interval of length > 0). Therefore the result is not of type `AccumBounds` which is expected to point to a subset of extended reals. It would be possible to define "complex accumulation bounds" but those have not been implemented. So I think that all `b**e` where `e` is an instance of `AccumBounds` and `b` or `b.min` is negative should be left unevaluated for now.

Reply:
I am abusing the bounds notation, right? What I really mean is `{-1, 1}` are the only real answers. Perhaps all this logic is better applied to SetExpr. (Or, as you indicate, just let negative bases be unevaluated or raise an error that 'negative bases have finite solutions only for integer exponents'.)